PacketTunnelProvider/NetworkProtection/NetworkProtectionPacketTunnelProvider.swift

```diff
@@ -479,28 +479,33 @@ final class NetworkProtectionPacketTunnelProvider: PacketTunnelProvider {
         APIRequest.Headers.setUserAgent(DefaultUserAgentManager.duckDuckGoUserAgent)
     }
 
+    deinit {
+        memoryPressureSource?.cancel()
+        memoryPressureSource = nil
+    }
+
+    private var memoryPressureSource: DispatchSourceMemoryPressure?
+    private let memoryPressureQueue = DispatchQueue.init(label: "com.duckduckgo.mobile.ios.NetworkExtension.memoryPressure")
```

Comment:
Nitpicky and quite optional: would really really consider removing the `init` to make the code less verbose.